src/inverse-kinematics/src/InverseKinematics.cpp

```diff
@@ -289,6 +291,17 @@ namespace iDynTree {
 
         iDynTree::Axis projectionPlaneXaxisInAbsoluteFrame(xAxisOfPlaneInWorld,originOfPlaneInWorld);
         iDynTree::Axis projectionPlaneYaxisInAbsoluteFrame(yAxisOfPlaneInWorld,originOfPlaneInWorld);
+	
+	// Initialize the COM's projection direction in such a way that is along the lien perpendicular to the xy-axes of the World
+	iDynTree::Direction zAxis;
+	toEigen(zAxis) = toEigen(xAxisOfPlaneInWorld).cross(toEigen(yAxisOfPlaneInWorld));
+	
+	Vector3 zAxisOfPlaneInWorld;
+	zAxisOfPlaneInWorld(0) = zAxis.getVal(0);
+	zAxisOfPlaneInWorld(1) = zAxis.getVal(1);
+	zAxisOfPlaneInWorld(2) = zAxis.getVal(2);
+		
+	IK_PIMPL(m_pimpl)->m_comHullConstraint.setProjectionAlongDirection(zAxisOfPlaneInWorld);
```

Comment:
I think we can directly call `IK_PIMPL(m_pimpl)->m_comHullConstraint.setProjectionAlongDirection(zAxis)` here.